src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -1,11 +1,84 @@
 import React from 'react';
 import FrontPageMain from '.';
 import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
+import frontPageDataIgbo from '../../../../data/prod/igbo/frontpage';
 import frontPageDataPidgin from '../../../../data/prod/pidgin/frontpage';
+import frontPageDataYoruba from '../../../../data/prod/yoruba/frontpage';
+
+const igboConfig = {
+  brandName: 'BBC News Ìgbò',
+  translations: {
+    home: 'Akụkọ',
+  },
+};
+
+const pidginConfig = {
+  brandName: 'BBC News Pidgin',
+  translations: {
+    home: 'Home',
+  },
+};
+
+const yorubaConfig = {
```

Comment:
Apologies, but I dont think we need 3 services here, 1 service should be fine as there is no specific logic in the code that changes the behavior between them

Reply:
Done. Let me know if we should add more dummy data to the `igboConfig` object.